src/stores/models/Oracle.test.js

```diff
@@ -0,0 +1,206 @@
+import { observable } from 'mobx';
+import { Token, Phases, OracleStatus } from 'constants';
+import Oracle from './Oracle';
+import { decimalToSatoshi } from '../../helpers/utility';
+
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+const { QTUM, BOT } = Token;
+const { BETTING, VOTING, RESULT_SETTING } = Phases;
+
+describe('models/Oracle', () => {
+  let oracle;
+  const addr = {
+    address: 'qSu4uU8MGp2Ya6j9kQZAtizUfC82aCvGT1',
+    qtum: 2000,
+    bot: 100,
+  };
+  const app = observable({
+    sortBy: 'ASC',
+    wallet: { addresses: [addr] },
+    global: { syncBlockNum: 0 },
+    refreshing: false,
+    ui: { location: 0 },
+  }); // mock the appstore
+  beforeEach(() => {
+    oracle = null;
+  });
+
+  /** all following test cases target specifically to the mock backend, eg. network/graphql/__mocks__/queries.js */
+  it('Constructor confirmed betting', async () => {
```

Comment:
wrap constructor tests in describe block

Reply:
nit: write test names like a sentence, `it 'constructs a confirmed betting Oracle'`